typings/index.d.ts

```diff
@@ -3138,23 +3213,48 @@ declare module 'discord.js' {
 
   type MessageAdditions = MessageEmbed | MessageAttachment | (MessageEmbed | MessageAttachment)[];
 
+  interface MessageActionRowOptions extends BaseMessageComponentOptions {
+    type: 'ACTION_ROW';
+    components?: MessageComponentResolvable[];
+  }
+
   interface MessageActivity {
     partyID: string;
     type: number;
   }
 
+  interface MessageButtonOptions extends BaseMessageComponentOptions {
```

Comment:
Is it intended that this does not have a `type` property of `'BUTTON' | MessageComponentTypes.BUTTON`?